samples/app-auth/nodejs/.env

```diff
@@ -0,0 +1,9 @@
+MicrosoftAppId=
```

Comment:
Why we delete it?
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/59052547/612b2ce0-a904-4827-a335-8c2ffa627d26)

Reply:
Not required, we will add toolkit scripts in another PR